uaclient/entitlements/livepatch.py

```diff
@@ -67,6 +70,9 @@ def configure_livepatch_proxy(
     :@param livepatch_retries: Optional list of sleep lengths to apply between
                                snap calls
     """
+    if http_proxy or https_proxy:
+        print(status.MESSAGE_SETTING_SERVICE_PROXY.format(service="livepatch"))
```

Comment:
We may want to use class attr here `LivepatchEntitlement.title` as the enable/disable messaging sources that value for a Capitalized value for the service name.  This way if we change `LivepatchEntitlement.title` in the future we won't have to find this text and adapt it later